client/src/use/useTextLayer.js

```diff
@@ -9,7 +9,7 @@ export default function useTextLayer({
   const vuetify = inject('vuetify');
 
   const textData = computed(() => {
-    if (!filteredDetections.value.length) {
+    if (filteredDetections.value.length === 0) {
       return null;
```

Comment:
Is this not supposed to also return `[]`?

Reply:
I would think so, I need to check some of the other files (useTrackFilterControls and others).  I'm assuming we would want to try to maintain returning the same type in the future.  Especially if the component prop is expecting that type.